src/Mado/QueryBundle/Objects/RequestOptions.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Mado\QueryBundle\Objects;
+
+use Symfony\Component\HttpFoundation\Request;
+
+class RequestOptions
```

Comment:
This object is very similar to QueryBuilderOptions. Except for the constructor. They have in common tha parameters. QBO was born to work without requests. Both should have same interface. Whenever we use them, we should play with `BuilderOptionsInterface` or something similar, instead of with QBO or this new RO. We have to talk about this ^_^